packages/orbit-components/src/Popover/index.tsx

```diff
@@ -188,7 +188,7 @@ const Popover = ({
         tabIndex={0}
         onClick={handleClick}
         onKeyDown={handleKeyDown<HTMLDivElement>(handleClick)}
-        aria-controls={overlayId}
+        aria-controls={id || popoverId}
```

Comment:
I fixed this as it was an easy fix and it's mentioned in the doc so I didn't want us to mention something broken.